app/api/v2/handlers/schedule_api.py

```diff
@@ -0,0 +1,129 @@
+    @aiohttp_apispec.request_schema(ScheduleSchema(partial=True, only=['schedule', 'task.autonomous', 'task.state',
+                                                                       'task.obfuscator']))
+    @aiohttp_apispec.response_schema(ScheduleSchema, description='The response is a dump of the newly '
+                                                                 'updated Schedule object.')
+    async def update_schedule(self, request: web.Request):
+        schedule = await self.update_object(request)
+        return web.json_response(schedule.display)
+
+    @aiohttp_apispec.docs(tags=['schedules'], summary='Replace Schedule',
+                          parameters=[{
+                              'in': 'path',
+                              'name': 'id',
+                              'schema': {'type': 'string'},
+                              'required': 'true',
+                              'description': 'UUID of the Schedule to be retrieved.'
+                          }],
+                          description='Use fields from the ScheduleSchema in the request body '
+                                      'to replace an existing Schedule or create a new Schedule.')
+    @aiohttp_apispec.request_schema(ScheduleSchema(partial=True, exclude=['id']))
+    @aiohttp_apispec.response_schema(ScheduleSchema, description='The response is a dump of the newly '
+                                                                 'replaced Schedule object.')
+    async def create_or_update_schedule(self, request: web.Request):
+        schedule = await self.create_or_update_object(request)
+        return web.json_response(schedule.display)
+
+    @aiohttp_apispec.docs(tags=['schedules'], summary='Delete Schedule',
+                          parameters=[{
+                              'in': 'path',
+                              'name': 'id',
+                              'schema': {'type': 'string'},
+                              'required': 'true',
+                              'description': 'UUID of the Schedule to be retrieved.'
+                          }],
+                          description='Deletes a Schedule object from the data service.')
+    @aiohttp_apispec.response_schema(ScheduleSchema,
+                                     description='Returns HTTP 204 No Content status code if '
+                                                 'Schedule is deleted successfully.')
+    async def delete_schedule(self, request: web.Request):
+        await self.delete_object(request)
+        return web.HTTPNoContent()
+
+    '''Overridden Methods'''
+
+    async def create_object(self, request: web.Request):
+        data = await request.json()
+        await self._error_if_object_with_id_exists(data.get(self.id_property))
+        access = await self.get_request_permissions(request)
+        operation = await self._api_manager.validate_and_setup_task(data['task'], access)
```

Comment:
let's put lines 118-120 into a helper method since they're repeated exactly in the `create_or_update_object` method